### PR TITLE
Add a basic tutorial

### LINKED ISSUE #108: Add a basic tutorial to explain user the functioning of app
Add a basic tutorial explaining user the functionality of app using material showcase.

### PR DESCRIPTION
**Pull request by mursyidah**
# Description

I already add a basic tutorial explaining user the functionality of app. But i only add 4 tutorial only. I dont know whether its enough for you or not. I just get the image from your Github. This is the example of introduction for the apps:

screen shot 2018-10-06 at 1 57 36 pm
 

Fixes #108 

## Type of change
Just put an x in the [] which are valid.
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: app/src/main/java/swati4star/createpdf/activity/MainActivity.java [hunk 1]
```diff
@@ -95,41 +94,45 @@ protected void onCreate(Bundle savedInstanceState) {
...
-                break;
-            case ACTION_MERGE_PDF:
-                fragment = new MergeFilesFragment();
-                setDefaultMenuSelected(2);
-                break;
-            default:
-                // Set default fragment
-                fragment = new HomeFragment();
-                break;
+        if (getIntent() != null && getIntent().getAction() != null ) {
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/activity/MainActivity.java]: Why are these changes needed?

File: app/src/main/java/swati4star/createpdf/activity/WelcomeActivity.java [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+        mViewPager = (ViewPager) findViewById(R.id.view_pager);
+        mDotsLayout = (LinearLayout) findViewById(R.id.layoutDots);
+        mBtnSkip = findViewById(R.id.btn_skip);
+
+
+
+        // layouts of all welcome sliders
+        // add few more layouts if you want
+        mLayouts = new int[]{
+                R.layout.fragment_intro_1,
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/activity/WelcomeActivity.java]: Can you add more verbose names to these layout files.
These are not giving much information on what these are.

File: app/src/main/java/swati4star/createpdf/activity/WelcomeActivity.java [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+        mViewPager.setAdapter(mViewPagerAdapter);
+        mViewPager.addOnPageChangeListener(viewPagerPageChangeListener);
+
+        mBtnSkip.setOnClickListener(view -> {
+            Intent intent = new Intent(getApplicationContext(), MainActivity.class);
+            startActivity(intent);
+        });
+    }
+
+    private void addBottomDots(int currentPage) {
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/activity/WelcomeActivity.java]: add comments to the newly added function

File: app/src/main/java/swati4star/createpdf/activity/MainActivity.java (hunk 1 above)
Comment (mursyidah) [reply app/src/main/java/swati4star/createpdf/activity/MainActivity.java]: because getIntent().getAction() returns a null pointer exception when the activity is opened, so i  make a few change to avoid the error.

File: app/src/main/java/swati4star/createpdf/activity/WelcomeActivity.java (hunk 2 above)
Comment (mursyidah) [reply app/src/main/java/swati4star/createpdf/activity/WelcomeActivity.java]: is it like R.layout.fragment_merge_pdf_step ?

File: app/src/main/java/swati4star/createpdf/activity/WelcomeActivity.java (hunk 3 above)
Comment (mursyidah) [reply app/src/main/java/swati4star/createpdf/activity/WelcomeActivity.java]: alright noted

File: app/src/main/java/swati4star/createpdf/activity/WelcomeActivity.java (hunk 2 above)
Comment (Swati4star) [reply app/src/main/java/swati4star/createpdf/activity/WelcomeActivity.java]: Yes. Something like this looks good.

File: app/src/main/java/swati4star/createpdf/activity/MainActivity.java (hunk 1 above)
Comment (Swati4star) [reply app/src/main/java/swati4star/createpdf/activity/MainActivity.java]: If this is the case, then you can just add a null check.

**Comment by Swati4star** [comment]
Looks great! @mursyidah 👍 
Thanks :)
